package.json

```diff
@@ -109,9 +114,11 @@
     "eslint-import-resolver-typescript": "^3.5.2",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-lernfair-lint": "./linter/",
+    "jest": "^29.5.0",
```

Comment:
I've added jest to run unit tests, but I'm open in case anyone prefers a different framework.